### ISSUE TITLE
Must issue a  STARTTLS command first when sending email

### ISSUE THREAD

**Issue by Amishman** [alekdavis/PlexBackup#32]
I am able to send an email through Gmail with the less secure app turned on. When I try to send an email with 2 steps verification turned on, I receive the following error message.

![image](https://user-images.githubusercontent.com/5334426/83958040-3f7e4c80-a83b-11ea-8e87-0b42290157c6.png)

I use the code generated by Gmail as my new password in my credentials.xml. It does not give me an error message about the credentials.

**Comment by alekdavis** [comment]
Based on the answer posted here (https://stackoverflow.com/questions/17281669/using-smtp-gmail-and-starttls/21909926#:~:text=Gmail%20exposes%20port%20465%20for,then%20switches%20to%20SSL%2FTLS.), try changing port in the configuration file (or via command line) to 465 (by default, it uses port 587). See if it helps.

**Comment by Amishman** [comment]
I changed the port to 465 and now it times out.

![image](https://user-images.githubusercontent.com/5334426/84133052-a3a83880-aa14-11ea-970a-09e8a1d54e6b.png)

**Comment by alekdavis** [comment]
Can you troubleshoot it from PowerShell command line? Forget about the PlexBackup script. Try something like an example here (include the port flag if the `-Port` command-line switch):

https://stackoverflow.com/a/54129858/52545

**Comment by Amishman** [comment]
I finally got PlexBackup to work properly. Originally, I had the "To" and "From" as the same Gmail account. When I tried the example you provided, I used the original Gmail account as the "From" and another Gmail account as the "To". When I used port 465, I got the time out error but 587 worked. I then changed the information to match in PlexBackup.ps1.json and it worked. I'm guessing Gmail didn't like sending/receiving a 2 step verification email to the same account.

Thank you for your time and script. It is much appreciated.
